site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -1373,5 +1373,80 @@ public class MyAnnotatedService {
     public HttpResponse update() { ... }
 }
 ```
+
+## Specifying the default service name
+
+The default name of an annotated service which served a request is the innermost class name.
+The service name is often used as a meter tag or distributed trace's span name.
+If you want to customize the default service name, then you can override the name by annotating
+a class or method with <type://@ServiceName> like the below:
```

Comment:
A service name is a human-readable string that is often used as a meter tag or distributed trace's span name. By default, an annotated service uses its class name as its service name. You can override it by annotating a client or method with ... like the following: